Flixter/MovieViewController.m

```diff
@@ -38,6 +38,7 @@ - (void)viewDidLoad {
     [self.tableView insertSubview:refreshControl atIndex:0];
 
     [self networkRequest];
+    NSLog(@"%@", self.movieData);
```

Comment:
I should explain why this drives me nuts. 
If you do this in an app you're working on with multiple people. At some point you will want to use the logs to debug something. You will add a statement but it will be swallowed into the sea of log statements. So you'll add something like `NSLog(@"ANGELINA ***: %@", self.movieData);` and then filter the logs. But then you'll miss something important like a log statement from a system library that's being deprecated, or a broken constraint that will make things look bad on some device sizes.

tldr; I promise that leaving log statements in is a bad habit that will come back to bite you at some point.